### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by wballoni** [angular-redux/platform#114]
### This is a...

- [X] feature request
- [ ] bug report
- [ ] usage question

### What toolchain are you using for transpilation/bundling?

- [X] @angular/cli
- [ ] Custom @ngTools/webpack
- [ ] Raw `ngc`
- [ ] SystemJS
- [ ] Rollup
- [ ] Other

### Additional Notes:
Not really a feature but a general question: Is this project going to be supported going forward, or is it dead? I understand managing such a project is a big commitment, but I am starting to see rust growing, that is documented in the issues, and no apparent work to address it.

If one of the maintainers can give the user community a heads-up on this it would be great, either way. Thanks.

**Comment by jesse-deboer** [comment]
I think the project is indeed dead, would be nice if a maintainer would let us know if it will be deprecated.

**Comment by masternone** [comment]
Maybe the solution is to not get one of the original maintainers to relinquish control but for someone who is commenting here to fork this project and we start maintaining the the fork.

**Comment by wbuchwalter** [comment]
Sorry for the delayed response, I haven't contributed to angular-redux in years so I am not monitoring issues.
It does look like this repo is not actively maintained anymore indeed.

@smithad15 @e-schultz Can you confirm?
If the answer is yes or there is no response from the most recent maintainers after a reasonable delay (say in a week) I am glad to add people as contributors if some of you are willing, otherwise I will mark the repository as deprecated.

**Comment by wbuchwalter** [comment]
Since there has been no answer we can assume that no one is maintaining this project anymore.

If some of you are willing to contribute to this repo I can add you as collaborator.
In the meantime I'll go ahead and clearly mark this repository as inactive for now.

**Comment by smithad15** [comment]
Correct, I have not been able to find the time to maintain this package, nor am I particularly close to Angular development anymore. Happy to help facilitate a transition, but I also haven’t seen many folks who want to contribute through PRs as yet. 

The good news is that the core of the package has been incredibly stable over the past couple of major releases and other than an npm dependency warning, should be dependable for some time to come. The places that require more attention are forms (some functionality is currently broken) and other more advanced feature sets. 

From what I have seen, the need for this package has dwindled over time with the Google team backing the active development of NgRX. In fact, I know of people who have successfully migrated off of this package towards NgRX without too much fuss and the primary APIs are very similar. If, however, you are maintaining an application that has a shared Redux state between multiple frameworks, this is still the best solution for the Angular portion of the Redux bindings. 

If someone if willing and able to take maintenance of this project on, then I would be happy to help transition what knowledge I currently know to them, but until that time please consider this repo not actively maintained.

**Comment by baart1989** [comment]
Thanks for creating this lib!

As repo is not maintained anymore, we have been considering migrating to NgRX.
But it seemed for me so big overhead. 

As in company we were using angular-redux with redux-observable.
Moving pure js/ts epics to NgRX effect's seemed for me biggest overhead, as it would require lots of changes.

As we are using only small portion of angular-redux (and all selectors are functions), I have created small connector between angular and redux.

In case anyone would encounter similar problem, here is [gist](https://gist.github.com/baart1989/a8d75be7846fa6d6521ec7821b7a1810) of it.

**Comment by ambientlight** [comment]
> If someone if willing and able to take maintenance of this project on, then I would be happy to help transition what knowledge I currently know to them, but until that time please consider this repo not actively maintained

@smithad15: we are using angular-redux internally and I know its codebase a bit, and I also looking into its integration with logux at https://github.com/logux/logux/issues/18 since we rely heavily on angular-redux. I would like to volunteer to main this.

**Comment by rplotkin** [comment]
> Since there has been no answer we can assume that no one is maintaining this project anymore.
> 
> If some of you are willing to contribute to this repo I can add you as collaborator.
> In the meantime I'll go ahead and clearly mark this repository as inactive for now.

Please add me as a collaborator. I'm a software architect, I use and rely upon this library, and I'm happy to help maintain it. @wbuchwalter

**Comment by rplotkin** [comment]
@wbuchwalter @smithad15 I am happy to help maintain this project. If you can add me as a collaborator, I can start with basic maintenance like Ng11 updates.

**Comment by wbuchwalter** [comment]
@rplotkin Unless @smithad15 can chime here to help with that, I think it would be best for you to start a fork as I don't have access to all the npm registries etc.
If you do start a fork, let me know here and I'll mark this repository as deprecated and point to your repo in the README.

**Comment by rplotkin** [comment]
@wbuchwalter this is a good idea. I'll do this, and set up CI and such. Once everything is passing I will post again

**Comment by rplotkin** [comment]
@wbuchwalter Just an update, I do have CI working, but the lerna monorepo structure and example-app are causing a lot of issues as I try to get things running and upgrading locally.  I am going to spike an nx monorepo structure, which I think will resolve upgradability. My plan is to release a v11 compatible with ng11 (and then keep parity in v numbers with Angular going forward).  I'll ensure 10 is compatible with 7/8/9/10, if possible, or otherwise do a 10.7, 10.8, 10.9, and 10.10.

I can get this to publish to my npm, but it would be ideal if, once I have that working, that I could take over the npmjs @angular-redux handle.  If that ends up not possible, then perhaps calling it @angular-redux-2/platform (or something like that) would be better than having it associated with my name.  Open to ideas.

**Comment by wbuchwalter** [comment]
@rplotkin I do not have access to the angular-redux npmjs handle as my involvement with this project predated it's creation.
So unless one of the admin of that handle step up here you will have to create a new one.

**Comment by smithad15** [comment]
Hey @rplotkin. Glad to hear you're game for taking over help with the library. If you want to get in touch with me on Twitter (@smithad15), we can work out how best to move forward 🙂

**Comment by HarelM** [comment]
I would be very interested to know how this was resolved.
My code base is using this library and I would hate to invest in migrating instead of invest in helping here.
Please let me know what you decided, hopefully here, and what are the next steps.
I'm in the middle of upgrading to Ng11 (from 8) and I would like to have the npm install warnings removed...

**Comment by Canos** [comment]
@HarelM any new about this migration?
Can we use the project in ng11?

**Comment by HarelM** [comment]
Since I did not receive any feedback I decided to take the relevant code and integrate it in my project.
This is less than ideal but I needed a long term solution...
See here:
https://github.com/IsraelHikingMap/Site/tree/master/IsraelHiking.Web/src/application/reducers/infra
If someone got the npm keys in order to push a new package I'd be happy to help out on that fork...

**Comment by Garefild** [comment]
Hey, I'm trying to maintain this library.
https://github.com/angular-redux2/store

> 
> 
> Since I did not receive any feedback I decided to take the relevant code and integrate it in my project.
> This is less than ideal but I needed a long term solution...
> See here:
> https://github.com/IsraelHikingMap/Site/tree/master/IsraelHiking.Web/src/application/reducers/infra
> If someone got the npm keys in order to push a new package I'd be happy to help out on that fork...

**Comment by HarelM** [comment]
@Garefild thanks for the info!
I'll give it a go once I have a few minutes to do the switch.
Thanks for taking the time to do this!
Would you be interested in integrating the following file into your library?
https://github.com/IsraelHikingMap/Site/blob/main/IsraelHiking.Web/src/application/reducers/infra/decorators/reducer-action-decorator.ts
I use it to define classes and then use them as reducers, as can be seen here for example:
https://github.com/IsraelHikingMap/Site/blob/main/IsraelHiking.Web/src/application/reducers/layers.reducer.ts

**Comment by Garefild** [comment]
@HarelM 
Thanks, I'll do it later or on the weekend.

**Comment by JianMeng** [comment]
What changes need to be make if let angular-redux/store  support angular 13 or 14?

**Comment by HarelM** [comment]
Use another library instead of this one:
angular-redux2/store or something similar which is currently maintained...

**Comment by baart1989** [comment]
Anyone having problem with this library, following file - https://gist.github.com/baart1989/a8d75be7846fa6d6521ec7821b7a1810 allowed me to get rid off 3rd party dependency, and still use redux in my app. Much easier then moving to any other lib, or try to maintain existing ones :)

**Comment by HarelM** [comment]
The above library I mentioned is well maintained, I encourage to use it.
While using a single file is probably a quick an elegant solution I'm not sure how easy it is to mock and test this, let alone understand how to migrate and use it.
I haven't tried, so I might be super wrong here... 😄
